src/main/java/org/ssu/edu/teachua/db/entities/Category.java

```diff
@@ -0,0 +1,40 @@
+package org.ssu.edu.teachua.db.entities;
+
+import lombok.Data;
+
+import org.ssu.edu.teachua.db.annotation.Column;
+import org.ssu.edu.teachua.db.annotation.ManyToMany;
+import org.ssu.edu.teachua.db.annotation.TableDB;
+
+import java.util.Set;
+
+@TableDB(name = "categories")
+@Data
+public class Category extends Entity {
+
+    @Column(name = "background_color")
+    public String backgroundColor;
+
+    @Column(name = "description")
+    public String description;
+
+    @Column(name = "name")
+    public String name;
+
+    @Column(name = "sortby")
+    public Integer sortBy;
+
+    @Column(name = "tag_background_color")
+    public String tagBackgroundColor;
+
+    @Column(name = "tag_text_color")
+    public String tagTextColor;
+
+    @Column(name = "url_logo")
+    public String urlLogo;
+
+    @ManyToMany(foreignTable ="clubs", tableForManyToMany = "club_category",
+            mainColumnDB = "category_id", foreignColumnDB = "club_id")
+    private Set<Club> club;
```

Comment:
![image](https://user-images.githubusercontent.com/86326168/225592560-a070c35f-f7ac-4abc-bd95-f6e344d3f1ba.png)
I don't know if we need @ ManyToMany annotation as @ ManyToMany = @ ManyToOne and @ OneToMany.
But in other way, your implementation is also good. We need clarifications from Liubomyr and  Ilya : )

Reply:
Ilya helped me with this annotation, so we have approving by him))))

Reply:
O, nice !